docs/faqs/contributor-faq.md

```diff
@@ -0,0 +1,218 @@
+Contributor FAQ
+===============
+
+The following contains answers to typical questions from contributors about OpenMS.
+
+## General
+
+The following section provides general information to new contributors.
+
+### I am new to OpenMS. What should I do first?
+
+* Check out the development version of OpenMS (see website).
```

Comment:
let's replace
> (see website)
w/
, see OpenMS [releases and installers]\(downloads/releases.md\).